src/app/seamly2d/dialogs/configpages/preferencesconfigurationpage.cpp

```diff
@@ -120,20 +107,13 @@ PreferencesConfigurationPage::PreferencesConfigurationPage(QWidget *parent)
     });
 
     // set default pattern making system
-    index = ui->systemCombo->findData(qApp->Seamly2DSettings()->GetPMSystemCode());
+    int index = ui->systemCombo->findData(qApp->Seamly2DSettings()->GetPMSystemCode());
     if (index != -1)
     {
         ui->systemCombo->setCurrentIndex(index);
     }
-    //---------------------- Send crash reports
-    //ui->sendReportCheck->setChecked(qApp->Seamly2DSettings()->GetSendReportState());
-    //ui->description = new QLabel(tr("After each crash Seamly2D collects information that may help us fix the "
-    //                                "problem. We do not collect any personal information. Find more about what %1"
-    //                                "kind of information%2 we collect.")
-    //                             .arg("<a href=\"https://wiki.seamly2d.com/wiki/UserManual:Crash_reports\">")
-    //                             .arg("</a>"));
 
-    //----------------------------- Pattern Editing
+    // Default opertsions suffixes
```

Comment:
Is 'opertsions' a typo?

Reply:
Definitely a typo... should be "operations"